src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -646,7 +646,15 @@
          Conversion is represented by a single BoundConversion.
          -->
     <Field Name="ConversionGroupOpt" Type="ConversionGroup" Null="allow"/>
-</Node>
+  </Node>
+
+  <!-- A special node for emitting the implicit conversion from an array (of T) to System.ReadOnlySpan<T> -->
+  <Node Name="BoundReadOnlySpanFromArray" Base="BoundExpression">
+    <!-- The type is required an instance of the well-known type System.ReadOnlySpan<T> -->
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="Operand" Type="BoundExpression"/>
+    <Field Name="Method" Type="MethodSymbol"/>
```

Comment:
>Method [](start = 17, length = 6)

nit: this method is the conversion operator used as fallback (when bytes are not in a good shape to be directly pointed at). Maybe add a comment or rename to "ConversionMethod"?